.github/workflows/release.yml

```diff
@@ -128,42 +149,59 @@ jobs:
               exit 1
           }
 
+          $selectedAsset = $null
+          Foreach ($asset in $selectedRelease.assets) {
+            if ($asset.name -eq "sauce-playwright-win.zip") {
```

Comment:
The windows runner is called `sauce-playwright-runner.zip` though 🤔

Reply:
@alexplischke  It was it discrepancy in the upload assets part. I uniformized it to have only one name.